backend/libraries/ballerina-core-python/tests/parsing/tuple_test.py

```diff
@@ -28,141 +29,161 @@
 
 def test_tuple_2_to_json() -> None:
     value = (1, 2)
-    serializer = tuple_2_to_json(int_to_json, int_to_json)
+    serializer = tuple_2_to_json(int32_to_json, int32_to_json)
     serialized = serializer(value)
-    assert serialized == {"kind": "tuple", "elements": [int_to_json(1), int_to_json(2)]}
+    assert serialized == {"discriminator": "tuple", "value": [int32_to_json(1), int32_to_json(2)]}
 
 
 def test_tuple_2_from_json() -> None:
-    serialized: Json = {"kind": "tuple", "elements": [int_to_json(1), int_to_json(2)]}
-    parser = tuple_2_from_json(int_from_json, int_from_json)
+    serialized: Json = {"discriminator": "tuple", "value": [int32_to_json(1), int32_to_json(2)]}
```

Comment:
sometimes you use the constant, sometimes you don't.

Reply:
it's because of the limited unpacking that I can't just put it everywhere, so it's easy to miss